cmd/istioctl/istioctl_test.go

```diff
@@ -99,3 +99,70 @@ func TestBogusExplicitKubeConfig(t *testing.T) {
 		t.Fatalf("Did not fail setting up client with bogus kubeconfig: %v", err)
 	}
 }
+
+func TestGet(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/dest-policy.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err != nil {
+		t.Fatalf("Could not create destination policy: %v", err)
+	}
+
+	file = ""
+	if err := getCmd.RunE(getCmd, []string{"destination-policy", "world-cb"}); err != nil {
+		t.Fatalf("Did not find world-cb %v", err)
+	}
+
+	if err := deleteCmd.RunE(deleteCmd, []string{"destination-policy", "world-cb"}); err != nil {
+		t.Fatalf("Could not delete world-cb: %v", err)
+	}
+
+	if err := getCmd.RunE(getCmd, []string{"destination-policy", "world-cb"}); err == nil {
+		t.Fatalf("Found world-cb after deletion ")
+	}
+}
+
+func TestNewGet(t *testing.T) {
+	rootSetup(t)
+
+	file = "testdata/dest-policy.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err != nil {
+		t.Fatalf("Could not create destination policy: %v", err)
+	}
+
+	file = ""
```

Comment:
Some of the `getCmd` code could be converted to table-driven tests (see https://github.com/golang/go/wiki/TableDrivenTests) to avoid duplication.